src/OSPSuite.Presentation/Presenters/Importer/InconsistenMoleculeAndMoleWeightException.cs

```diff
@@ -0,0 +1,10 @@
+﻿using OSPSuite.Assets;
+using System;
+
+namespace OSPSuite.Presentation.Presenters.Importer
+{
+   public class InconsistenMoleculeAndMoleWeightException : Exception
+   {
+      public InconsistenMoleculeAndMoleWeightException() : base(Error.InconsistenMoleculeAndMoleWeightException) { }
```

Comment:
You do not need to define your own exception all the time, especially if you are not catching them specifically

Reply:
It is catch now so we do need it.